programs/steward/src/instructions/idle.rs

```diff
@@ -14,6 +17,16 @@ pub struct Idle<'info> {
         bump
     )]
     pub state_account: AccountLoader<'info, StewardStateAccount>,
+
+    /// CHECK: Correct account guaranteed if address is correct
+    #[account(address = deserialize_stake_pool(&stake_pool)?.validator_list)]
+    pub validator_list: AccountInfo<'info>,
+
+    /// CHECK: Correct account guaranteed if address is correct
+    #[account(
+        address = get_stake_pool_address(&config)?
+    )]
+    pub stake_pool: AccountInfo<'info>,
```

Comment:
we added the validator list address to the config account so this isn't necessary

Reply:
a few other instructions where we can remove stake_pool as well